client/src/app/queries/sboms.ts

```diff
@@ -113,13 +121,20 @@ export const useFetchSbomsByPackageId = (
 ) => {
   const { data, isLoading, error, refetch } = useQuery({
     queryKey: [SBOMsQueryKey, "by-package", packageId, params],
-    queryFn: () => getSBOMsByPackageId(packageId, params),
+    queryFn: () =>
+      dataOf(
+        listRelatedSboms({
+          client,
+          path: { id: packageId },
+          query: { ...requestParamsQuery(params) },
+        })
+      ),
```

Comment:
The problem reported before still exists:

> This change is breaking the current UI.
> To reproduce it:
>- Go to the packages list page
>- Click on any package and you will be redirected to the package details page
>- Click on the `SBOMS` tab
>- Open your Browser dev tools and you will see a request like:
>```
>GET http://localhost:3000/api/v1/sbom/by-package
>```
>
>That is wrong as the expected request should be something like:
>```
>GET http://localhost:3000/api/v1/sbom/by-package?limit=10&offset=0&id=85901454-4c44-5572-b3b6->7dba8aefb3e3
>```

The "packageId" should be passed as a query parameter and not as a path parameter.